de.metas.salescandidate.base/src/main/java/de/metas/ordercandidate/api/IOLCandEffectiveValuesBL.java

```diff
@@ -194,13 +195,12 @@
 	I_M_Product getM_Product_Effective(I_C_OLCand olCand);
 
 	/**
-	 * Returns <code>C_UOM_ID</code> if <code>IsManualPrice='Y'</code> and <code>C_UOM_Internal_ID</code> otherwise.
-	 *
-	 * @param olCand
-	 * @return
+	 * Returns <code>C_UOM_ID</code> if <code>IsManualPrice='Y'</code> and {@link #getRecordOrStockUOMId(I_C_OLCand)} otherwise.
```

Comment:
```suggestion
	 * Returns {@link #getRecordOrStockUOMId(I_C_OLCand)} (i.e. the record's own UOM-ID) if <code>IsManualPrice='Y'</code> and <code>C_UOM_Internal_ID</code> (i.e. metasfresh's pricing-engine-based UOM-ID) otherwise.
```